### DISCUSSION TITLE
Missing Timeline Items when using actions/github-script or octokit/graphql-action

### DISCUSSION THREAD

**Discussion by daniesg** [community/community#25671] [Actions]
<p>I’m trying to get CrossReferencedEvents from Issues. If I use the the online GraphQL API Explorer (<a href="https://docs.github.com/en/graphql/overview/explorer" class="inline-onebox" rel="noopener nofollow ugc">Explorer - GitHub Docs</a>), I see the event. But if I use those 2 actions (actions/github-script and octokit/graphql-action) in a workflow, it’s like the event doesn’t exists.</p>
<p>For example, I’m running the query:</p>
<pre><code class="lang-auto">            query($nodeid:ID!) { 
              node(id: $nodeid) {
                ... on Issue {
                  id
                  number
                  labels(first: 100) {
                    nodes {
                      name
                    }
                  }
                  timelineItems(last: 100) {
                    nodes {
                      __typename
                      ... on CrossReferencedEvent {
                        createdAt
                        source {
                          ... on PullRequest {
                            id
                            state
                          }
                        }
                      }
                    }
                  }
                }
              }
            }
</code></pre>
<p>Using the online explorer, I see</p>
<pre><code class="lang-auto">        {
          "__typename": "LabeledEvent"
        },
        {
          "__typename": "AssignedEvent"
        },
        {
          "__typename": "AddedToProjectEvent"
        },
        {
          "__typename": "MilestonedEvent"
        },
        {
            "__typename": "CrossReferencedEvent",
            "createdAt": "...",
            "source": {
              "id": "...",
              "state": "MERGED"
            }
</code></pre>
<p>But if I use the 2 actions in a workflow, I only get</p>
<pre><code class="lang-auto">        {
          "__typename": "LabeledEvent"
        },
        {
          "__typename": "AssignedEvent"
        },
        {
          "__typename": "AddedToProjectEvent"
        },
        {
          "__typename": "MilestonedEvent"
        }
</code></pre>
<p>Am I missing a preview or something?</p>

**Comment by daniesg** [comment, marked as answer]
<p>Figured it out. Needed to use a custom access token since the cross referenced pull request is in a different repo. Default github secret is only for current repo.</p>

**Reply by InvisCat** [reply]
yeah, it is necessary to switch to **classic** PAT to solve this issue.... (But it is not clearly mentioned in the documentation, which can sometimes be confusing)
